client/components/Pagination.vue

```diff
@@ -1,15 +1,24 @@
 <template>
   <div>
-    <v-pagination v-model="page" :length="15" :total-visible="4" />
+    <v-pagination
+      v-model="page"
```

Comment:
コンソール上に以下のエラーが表示されています。
```
[Vue warn]: Avoid mutating a prop directly since the value will be overwritten whenever the parent component re-renders. Instead, use a data or computed property based on the prop's value. Prop being mutated: "page"

found in

---> <Default1> at client/components/Pagination.vue
       <Default1> at client/pages/search.vue
         <Nuxt>
           <VApp>
             <Client/layouts/default.vue> at client/layouts/default.vue
               <Root>
```
`v-model`を使うことで、親から受け取った`page`の値を子コンポーネント上で上書きしてしまうため、 `value`を使ってください。
参考）https://qiita.com/simezi9/items/c27d69f17d2d08722b3a